pkg/shardddl/pessimism/ops.go

```diff
@@ -58,3 +59,13 @@ func DeleteInfosOperations(cli *clientv3.Client, infos []Info, ops []Operation)
 	_, rev, err := etcdutil.DoOpsInOneTxnWithRetry(cli, opsDel...)
 	return rev, err
 }
+
+// DeleteInfosOperationsThroughTask deletes the shard DDL infos and operations of a specified task in etcd.
+// This function should often be called by DM-master when deleting ddl meta data.
+func DeleteInfosOperationsThroughTask(cli *clientv3.Client, task string) (int64, error) {
```

Comment:
How about renaming `Through` to `For`? (if you accept this comment, update other `Through` too)

Reply:
changed to By